apps/emqx/src/emqx_schema.erl

```diff
@@ -1789,6 +1789,22 @@ fields("session_persistence") ->
                     desc => ?DESC(session_ds_last_alive_update_interval)
                 }
             )},
+        {"session_gc_interval",
+            sc(
+                timeout_duration(),
+                #{
+                    default => <<"10m">>,
+                    desc => ?DESC(session_ds_session_gc_interval)
```

Comment:
`rel/i18n/emqx_schema.hocon` should be updated?

Reply:
Strange that spellcheck didn't complain.  I noticed that other PRs added options here but didn't add translations there.  :thinking: